src/components/ActivityForm.js

```diff
@@ -10,40 +11,40 @@ export default class ActivityForm extends React.Component {
       room: props.activity ? props.activity.room : '',
       teacher: props.activity ? props.activity.teacher : '',
       error: ''
-      //TODO: when you will use prop-types you will be able to improve this place ;)
+      //TODO: when you will use prop-types you will be able to improve this place ;)
     };
-  };
+  }
 
-  onNameChange = e => this.setState({name: e.target.value});
+  onNameChange = e => this.setState({ name: e.target.value });
 
-  onTeacherChange = e => this.setState({teacher: e.target.value});
+  onTeacherChange = e => this.setState({ teacher: e.target.value });
 
-  onRoomChange = e => this.setState({room: e.target.value});
+  onRoomChange = e => this.setState({ room: e.target.value });
 
-  onClassNoChange = (e) => {
+  onClassNoChange = e => {
     const classNo = e.target.value;
     if (!classNo || classNo.match(/^[1-9]\d*$/)) {
-      this.setState({classNo});
-    };
+      this.setState({ classNo });
+    }
   };
 
-  onDayChange = (e) => this.setState({day: e.target.value});
+  onDayChange = e => this.setState({ day: e.target.value });
 
-  onSubmit = (e) => {
+  onSubmit = e => {
     e.preventDefault();
     const { name, day, classNo, room, teacher } = this.state;
     if (!name || !day || !classNo) {
-      this.setState({error: 'Please provide Activity Name, Day and Class Number!'});
+      this.setState({ error: 'Please provide Activity Name, Day and Class Number!' });
```

Comment:
What if I provide Activity Name and Day, but not Class Number? For user it will be confusing ;) but this is to handle by separate PR!